src/messages/components/ConversationCompose.vue

```diff
@@ -104,13 +107,21 @@ export default {
   },
   data () {
     return {
-      message: (this.value) || '',
+      message: (this.value) || localStorage[this.conversation] || '',
       hasFocus: false,
+      key: this.conversation,
```

Comment:
This is not reactive: if conversation ID gets loaded lazily (i.e. after this component initializes) or changes, `key` will be invalid.

Reply:
agreed I switched that to a computed prop